CMakeLists.txt

```diff
@@ -228,6 +246,15 @@ else()
     message(STATUS "libavif: libyuv not found; libyuv-based fast paths disabled.")
 endif()
 
+find_package(WebP) # not required
+if(TARGET WebP::sharpyuv)
```

Comment:
`FIndWebP.cmake` is not shipped w/ CMake AFAICT (nor does the standard libwebp package supply the CMake config files), so please supply one.

Reply:
Sorry, [the missing CMake config file could be just a MSYS2 packaging issue](https://packages.msys2.org/package/mingw-w64-ucrt-x86_64-libwebp?repo=ucrt64), will investigate (it seems it is in the source tree for the 1.2.4 tag...)

Reply:
Ah, MSYS2 (and I suspect many other distros) still build and package libwebp using autotools, so it still might not be advisable to rely on this? Maybe also include a fallback searching via pkg-config?

Reply:
Thank you for pointing this out and investigating! I'll look into it.

Reply:
Thanks. Can you also update WebP's CMakeLists so the configure_file() commands for all the .pc files include "@ONLY" please?

Reply:
> Thanks. Can you also update WebP's CMakeLists so the [configure_file() commands for all the .pc files](https://github.com/webmproject/libwebp/blob/9b902cbacedd0fb19de5ce4475d7a8345d7fe9cf/CMakeLists.txt#L158) include `@ONLY` please (i.e. behaviour is on par w/ autotools)?

Thanks for pointing that out. I made the change in:
https://chromium-review.googlesource.com/c/webm/libwebp/+/3836718

Reply:
Done, I've added a Findlibsharpyuv.cmake that relies on pkg-config.